python/test/t_MethodOfMomentsFactory_std.py

```diff
@@ -66,16 +76,21 @@
 finiteLowerBound = [False, True]
 finiteUpperBound = [False, True]
 bounds = ot.Interval(lowerBound, upperBound, finiteLowerBound, finiteUpperBound)
-factory = ot.MethodOfMomentsFactory(distribution, bounds)
+factory = ot.MethodOfMomentsFactory(distribution, [3, 4], bounds)
 factory.setKnownParameter([a, b], [2, 3])
 factory.setOptimizationBounds(bounds)
-inf_distribution = factory.build(sample)
-print("estimated distribution=", inf_distribution)
+inf_dist = factory.build(sample)
+print("estimated distribution=", inf_dist)
+p_ref = [0.805158, 1.5, -0.783954, 2.42385]
+ott.assert_almost_equal(inf_dist.getParameter(), p_ref, 1e-2, 1e-2)
 
 # setKnownParameter+buildEstimator
-sample = ot.Normal(2.0, 1.0).getSample(10)
-factory = ot.MethodOfMomentsFactory(ot.WeibullMin())
+sample = ot.Normal(2.0, 1.0).getSample(size)
+factory = ot.MethodOfMomentsFactory(ot.Normal(), [1])
```

Comment:
Why the switch from WeibullMin to Normal ?

Reply:
no reason in particular, I added more tests and increased the size anyway